ios/FluentUI.Demo/FluentUI.Demo/Demos.swift

```diff
@@ -19,6 +19,7 @@ let demos: [(title: String, controllerClass: UIViewController.Type)] = [
     ("DateTimePicker", DateTimePickerDemoController.self),
     ("DrawerController", DrawerDemoController.self),
     ("HUD", HUDDemoController.self),
+	("IndeterminateProgressBar", IndeterminateProgressBarDemoController.self),
```

Comment:
Did you run the swift lint tool on the code? This still appears to be a tab instead of four spaces.